controllers/handlers/kubevirt.go

```diff
@@ -422,7 +441,7 @@ func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfigu
 
 	seccompConfig := getKVSeccompConfig()
 
-	networkBindings := getNetworkBindings(hc.Spec.NetworkBinding)
+	networkBindings := getNetworkBindings(hc.Spec.NetworkBinding, hc.Annotations)
```

Comment:
Optional: pass the whole HC CR, to simplify the function signature. Also relevant for `getKvFeatureGateList` and `getFeatureGateChecks`.

Reply:
same as https://github.com/kubevirt/hyperconverged-cluster-operator/pull/3600#discussion_r2194411194 i think